realm/realm-library/src/main/cpp/io_realm_internal_OsRealmConfig.cpp

```diff
@@ -346,7 +346,8 @@ JNIEXPORT jstring JNICALL Java_io_realm_internal_OsRealmConfig_nativeCreateAndSe
         if (!user) {
             JStringAccessor realm_auth_url(env, j_auth_url);
             JStringAccessor refresh_token(env, j_refresh_token);
-            user = SyncManager::shared().get_user(sync_user_identifier, refresh_token);
+            JStringAccessor access_token(env, j_access_token);
+            user = SyncManager::shared().get_user(sync_user_identifier, refresh_token, access_token);
```

Comment:
what happens if the access_token is expired?

Reply:
This logic will change once we connect using Stitch. Right now this is just a temporary workaround.